browser/mojSortableTable.ts

```diff
@@ -239,6 +239,53 @@ class PersistentSortOrder {
   }
 }
 
+class MOJFrontendTableSecondOrderSorter {
+  private secondOrderColumnNumber: number | undefined
+
+  constructor(private table: HTMLTableElement) {
+    // 'second-order-column' attribute set in server/utils/viewUtils.ts
+    const tableSecondOrderColumnAttr = this.table.getAttribute('second-order-column')
+    if (tableSecondOrderColumnAttr !== null) {
+      const parsedValue = parseInt(tableSecondOrderColumnAttr, 10)
+      if (!Number.isNaN(parsedValue)) {
+        this.secondOrderColumnNumber = parsedValue
+      }
+    }
+  }
+
+  secondOrderSort(rows: HTMLTableRowElement[], firstOrderColumnNumber: number): HTMLTableRowElement[] {
+    return rows.sort((rowA: HTMLTableRowElement, rowB: HTMLTableRowElement) => {
+      if (this.secondOrderColumnNumber !== undefined) {
```

Comment:
I wonder if it would be clearer to refactor this / break it up a bit to avoid the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) - I found the multiple nested `if`s a little tricky to read. You could do this by returning early, possibly breaking things out into private functions e.g. here you could remove one level of nesting by doing:
```
if (this.secondOrderColumnNumber === undefined) {
  return 0
}
```

Reply:
👍